tests/str/models.py

```diff
@@ -35,3 +35,7 @@ class InternationalArticle(models.Model):
 
     def __str__(self):
         return self.headline
+
+
+class Default(models.Model):
```

Comment:
I was planing to do so in #5613 as I discovered that while inlining prevents table creation it busts the global apps `_meta` cache and slow down things a bit further down the road. The appropriate way of inlining models require the isolation of declared model in a test specific registry. I will make sure to do the correct thing here however.